### ISSUE TITLE
Incompatible campaign/faction combinations.

### ISSUE THREAD

**Issue by SnappyComebacks** [dcs-liberation/dcs_liberation#2591]
### Affected versions

6.0.0

### Description

Create the campaign `Scenic Route` with Blufor as `United Arab Emirates 2005`, all other values left as default.  When at the Air Wing Configuration screen, there will be no blue squadrons prefilled, and no aircraft available in the `Add Squadron` modal.

![image](https://user-images.githubusercontent.com/74509817/204065333-5072a910-afaa-48ae-8b8e-0255786c0427.png)

### Save game and other files

No save game available.

**Comment by DanAlbert** [comment]
This campaign is only valid for factions that have a carrier or LHA. The description says as much. We could add a `requirements` field to campaigns, but I'm not sure if it'd be all that useful.

WDYT? Convert to FR, or just close as something we're unlikely to pursue?

**Comment by ColonelAkirNakesh** [comment]
Necroing this FR. For these campaigns (I also really like Fuzzle's Cyprus campaign and Starfire's Vectron's Claw), you *could* add a 'transfer from off-map' control point to populate the airwing then cheat-capture the first base and transfer their planes in.... this works for Vectron's Claw where you take Sukhumi first, but for some like Scenic Route and the Marianas and Cyprus ones, the air base is close to late-game and would pretty much break the flow of the campaign. My vote would close.

**Comment by DanAlbert** [comment]
I'll let @SnappyComebacks decide since it's something they might implement anyway :) it's be a good thing to have, I just doubt it'll ever make it to the top of _my_ todo list.

**Comment by ColonelAkirNakesh** [comment]
So what's the etiquette for updating maintained campaigns? I might do it myself since I want to fly Cyprus as Redfor (can't do SEAD in Su-33s) and wanna transfer in some Su-25Ts... Should I just add it myself, or should I reach out to Fuzzle first?

**Comment by Starfire13** [comment]
Speak to Fuzzle on Discord.

There is nothing stopping you from making a custom version of a campaign for your own use by the way. For example, I have a custom version of the Caen to Evreux campaign that I just copy-paste into my Liberation folder when I want to use it.

**Comment by DanAlbert** [comment]
Yeah, reach out first, but if there's no response I think have at it.

**Comment by ColonelAkirNakesh** [comment]
Fuzzle says he wants to keep it pure naval and that if someone wants a non-naval faction they can edit locally. Let's close this ^_^

**Comment by DanAlbert** [comment]
That's not what this FR is though. This FR is about flagging this sort of incompatibility in the UI.

**Comment by DanAlbert** [comment]
Well, it _could_ be that:

> WDYT? Convert to FR, or just close as something we're unlikely to pursue?

**Comment by ColonelAkirNakesh** [comment]
Ah ok. Sorry, I'm still learning and used the wrong terminology.. I think close as something unlikely to pursue.

**Comment by DanAlbert** [comment]
No worries :)
